### PR TITLE
[luci-interpreter] Replace large kernel builders collection

### PR DESCRIPTION
**Pull request by binarman**
This PR removes large collection of builders in one file with
separate files.

Related issue: #7288
draft : #7605

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

**Comment by binarman** [comment]
Waiting to merge SplitV PR's first: #7648 #7647

File: compiler/luci-interpreter/src/loader/KernelBuilder.cpp [hunk 1]
```diff
@@ -15,1261 +15,13 @@
  */
 
 #include "loader/KernelBuilder.h"
-
```
Comment (m-bronnikov) [inline compiler/luci-interpreter/src/loader/KernelBuilder.cpp:18]: @binarman, thank you =)
For reviewer: related to https://github.com/Samsung/ONE/pull/7614/files#r704665251

**Review by m-bronnikov — APPROVED** [review]
LGTM

File: compiler/luci-interpreter/src/loader/CMakeLists.txt [hunk 2]
```diff
@@ -10,6 +10,13 @@ set(SOURCES
     RuntimeToIR.h
     nodes/Builders.h)
 
+# include kernel specific builders
+macro(REGISTER_KERNEL NODE)
+  list(APPEND SOURCES "nodes/${NODE}.cpp")
+endmacro(REGISTER_KERNEL)
+include(${KERNEL_REGISTER_FILE})
+message("SOURCES" ${SOURCES})
```
Comment (m-bronnikov) [inline compiler/luci-interpreter/src/loader/CMakeLists.txt:18]: @binarman Do we need this msg here or better to make clean up PR? =)

Comment (m-bronnikov) [reply compiler/luci-interpreter/src/loader/CMakeLists.txt:18]: Personally, I would like to remove this line.

Comment (m-bronnikov) [reply compiler/luci-interpreter/src/loader/CMakeLists.txt:18]: Personally, I would like to remove this line =)

Comment (binarman) [reply compiler/luci-interpreter/src/loader/CMakeLists.txt:18]: this is a debug leftover, I noticed it too after merge =)
It should be removed, merge was not intentional

**Pull request merged**